docs/concepts.md

```diff
@@ -35,91 +46,102 @@ When there is late arriving data (data intended for 9:00 arriving >1 hr late at
+This is a much desired property for analytical workloads, which is predominantly read-heavy.
 
 Following illustrates how this works conceptually, when  data written into copy-on-write storage  and two queries running on top of it.
 
 
 {% include image.html file="hudi_cow.png" alt="hudi_cow.png" %}
 
 
-As data gets written, updates to existing file ids, produce a new version for that file id stamped with the commit and
-inserts allocate a new file id and write its first version for that file id. These file versions and their commits are color coded above.
+As data gets written, updates to existing file groups, produce a new slice for that file group stamped with the commit and
+inserts allocate a new file group and write its first version for that file id. These file slices and their commit instant times are color coded above.
 Normal SQL queries running against such dataset (eg: `select count(*)` counting the total records in that partition), first checks the timeline for latest commit
-and filters all but latest versions of each file id. As you can see, an old query does not see the current inflight commit's files colored in pink,
+and filters all but latest file slices of each file group. As you can see, an old query does not see the current inflight commit's files colored in pink,
 but a new query starting after the commit picks up the new data. Thus queries are immune to any write failures/partial writes and only run on committed data.
 
-The intention of copy on write storage, is to fundamentally improve how datasets are managed today on Hadoop through
+The intention of copy on write storage, is to fundamentally improve how datasets are managed today through
 
   - First class support for atomically updating data at file-level, instead of rewriting whole tables/partitions
   - Ability to incremental consume changes, as opposed to wasteful scans or fumbling with heuristical approaches
   - Tight control file sizes to keep query performance excellent (small files hurt query performance considerably).
 
 
-## Merge On Read
+## Merge On Read Storage
 
 Merge on read storage is a superset of copy on write, in the sense it still provides a read optimized view of the dataset via the Read Optmized table.
-But, additionally stores incoming upserts for each file id, onto a `row based append log`, that enables providing near real-time data to the queries
+But, additionally stores incoming upserts for each file group, onto a row based append log, that enables providing near real-time data to the queries
  by applying the append log, onto the latest version of each file id on-the-fly during query time. Thus, this storage type attempts to balance read and write amplication intelligently, to provide near real-time queries.
 The most significant change here, would be to the compactor, which now carefully chooses which append logs need to be compacted onto
```

Comment:
Can we replace `append logs` with `delta files` ? It is less confusing to the reader and build a single concept of 'delta' just like deltacommit is explained in previous sections.

Reply:
our code refers to it as logs, no? Delta logs?

Reply:
Delta logs sound fine to me.